aws/resource_aws_default_network_acl.go

```diff
@@ -243,7 +243,6 @@ func resourceAwsDefaultNetworkAclUpdate(d *schema.ResourceData, meta interface{}
 
 func resourceAwsDefaultNetworkAclDelete(d *schema.ResourceData, meta interface{}) error {
 	log.Printf("[WARN] Cannot destroy Default Network ACL. Terraform will remove this resource from the state file, however resources may remain.")
```

Comment:
I wonder if this is an issue that should be opened on core to have "no delete" resources?

Reply:
Are you looking for something like this? 😄 

https://godoc.org/github.com/hashicorp/terraform/helper/schema#Noop

Reply:
Actually this should probably just use: https://github.com/hashicorp/terraform/blob/1588fd2908a7e57390b78ab399b7ff5695c7e07b/helper/schema/resource.go#L566-L569

Reply:
I guess they are essentially the same in the case of a delete. Just semantics on them slightly different.

Reply:
The tfprovlint001 lint check is so we don't extraneously call `d.SetId("")`, using `RemoveFromState` would just retrigger that. Essentially, that function in helper/schema seems extraneous.